src/services/users/index.js

```diff
@@ -123,10 +132,45 @@ const inviteUser = async (email) => {
 	return signupInfo
 }
 
+const getNewTokens = async (email, currentRefreshToken) => {
+	const result = await  db.query(
+		`SELECT * FROM users 
+		WHERE email = $1 
+		AND "refreshToken" = $2
+		`,
+		[email, currentRefreshToken]
+	)
+	const userProfile = result.rows[0]
+
+	if (!userProfile) {
+		throw customError(refreshTokenIsInvalidError)
+	}
+
+	const user = userProfile
```

Comment:
Nice work so far

for this, we don't need to reassign here since nothing is really changing

Reply:
yes please, thank you